### DISCUSSION TITLE
🎉 New Video Alert: Your Next Steps After Patrick's Course! 🎥

### DISCUSSION THREAD

**Discussion by cromewar** [Cyfrin/foundry-full-course-cu#919] [Announcements]
Hello amazing Web3Education community! 🚀

First off, a massive shoutout to everyone who has already signed up for early access. Your enthusiasm and support fuel our mission, and we'll have more updates for you soon! 🌟

We've noticed that a number of you have been pondering on the question: "What do I do after completing Patrick's comprehensive course?". It's a journey, and the end of one chapter is just the beginning of another!

To help you navigate this phase, we've released a fresh video that addresses just that. Here's a sneak peek into what's covered:

Start Building: The best way to learn? Dive right in! We're giving you 5 personal portfolio ideas to kickstart your Web3 journey.
Join a Hackathon: Network, learn, embrace new tech, and yes, the potential to earn! Hackathons are a goldmine for growth and opportunities.
Job Hunting: It's time to put that knowledge to work. We've got tips and avenues for you to explore.
Delve into Security: Ever considered smart contract auditing? There's a vast ocean to explore here, and we'll tell you why it's worth diving into.
Keep Learning: The tech world moves fast. Stay updated, stay curious!

[Watch the Complete Video Here!](https://youtu.be/HRW7G0xo8yw)

![Miniatura para Youtube Aplicación de Dinero Sensacionalista Audaz Violeta y Verde](https://github.com/Cyfrin/foundry-full-course-f23/assets/20199180/cae5f216-2798-4129-a2c4-86d38d597554)

We've also integrated insights from the updated packages and addressed some of the most frequently raised issues from the community.

Remember, the journey of a thousand miles begins with a single step (or in our case, a single line of code 😉). We're here to guide, support, and cheer you on every step of the way!

Happy coding and learning! 💻❤️

**Comment by 0xusmanf** [comment]
Thank you so much ❤️, a much needed guideline. LFG 🚀

**Comment by theirrationalone** [comment]
Thankyou @cromewar !, For this Web3Verse Episode ❤

**Comment by shikhar229169** [comment]
Thank You ♥

**Comment by alfheimrShiven** [comment]
Thanks for sharing those interesting project ideas @cromewar 🙌🏾 . Will probably latch on to one. Cheers!
